hibernate-core/src/main/java/org/hibernate/boot/spi/SessionFactoryOptions.java

```diff
@@ -299,4 +299,6 @@ default int getQueryStatisticsMaxSize() {
 	default boolean isPostInsertIdentifierDelayableEnabled() {
 		return true;
 	}
+
+	boolean areJPACallbacksEnabled();
```

Comment:
Wondering if we need a default implementation here as it's SPI and it will be a micro?

The above method has one.

Reply:
good point!  I assumed this interface was not to be implemented, but I'm indeed not sure.